parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -33,6 +33,8 @@ if [[ "${TARGET_ENVIRONMENT,,}" == "${AZURE_STACK_ENV}"  ]]; then
     config_script_custom_cloud=/opt/azure/containers/provision_configs_custom_cloud.sh
     wait_for_file 3600 1 $config_script_custom_cloud || exit $ERR_FILE_WATCH_TIMEOUT
     source $config_script_custom_cloud
+    KUBERNETES_VERSION_NO_SUFFIX="${KUBERNETES_VERSION}"
+    KUBERNETES_VERSION=${KUBERNETES_VERSION}${AZURE_STACK_SUFFIX}
```

Comment:
this breaks the scenario : Azure public cloud + Ubuntu distro (requiring full install)

Reply:
ok, I will change the functions